suite-native/module-accounts/src/components/AccountsList.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { Box, Text } from '@suite-native/atoms';
+import { selectAccounts } from '@suite-common/wallet-core';
+
+import { AccountListItem } from './AccountListItem';
+
+export const AccountsList = () => {
+    const accounts = useSelector(selectAccounts);
+
+    if (accounts.length < 1) return <Text>No accounts found.</Text>;
```

Comment:
Not _too much_ :D It has been a convention for years. i have nothing again a plain JS approach in this case.

Reply:
Me too. I would go for this one to be honest.

Reply:
No, it's much harder to read and also it's very chaotic because you can write this is many different ways. This is no-go for me because you have like at least five ways how to check if array

```ts
if(accounts.lenght != 0)
if(accounts.lenght > 0)
if(accounts.lenght >= 1)
if(!accounts.lenght)
if(!accounts.lenght != 0)
if(!accounts.lenght > 0)
if(!accounts.lenght >= 1)
if(accounts.lenght)
if(accounts.lenght < 1)
if(accounts.lenght >= 1)
```

You have at least 10 ways (probably more) how to write check is something is empty or not empty. It's just super messy and implicit (you need to define how exactly check should happen). I bet you it will took a lot of time to go through it and say it which one does what. 

Now check this:
```
if(A.isEmpty(accounts)
if(A.isNotEmpty(accounts)
```


You only two ways how to do it. It's super explicit, so you don't need to care about how check itself is working and you can read your code like plain English and that is something what explicit code should do.

Reply:
Fixup commit coming